packages/components/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,108 @@
+import { Text } from "../../primitives/Text";
+
+export interface RadioProps extends RadioGroupItemProps {
+  /** The label of the radio. */
+  label: string;
+  /** The value of the radio. */
+  value: string;
+  /** Sets the state of the radio to disabled. */
+  disabled?: boolean;
+  /** Sets the state of the radio to disabled. */
+  hasError?: boolean;
+}
+
+/** A Radio is one button of a set of checkable buttons—known as radio buttons—where no more than one of the buttons can be checked at a time. */
+const Radio = React.forwardRef<HTMLButtonElement, RadioProps>(
+  ({ disabled = false, hasError = false, label, value, ...props }, ref) => {
+    const radioID = useUID();
+    return (
+      <Box.div alignItems="center" display="flex" gap="space30">
+        <Box.button
+          appearance="none"
+          aria-invalid={hasError}
+          as={RadioItem}
+          backgroundColor={{
+            _: "colorBackground",
+            hover: "colorBackgroundWeak",
+            checked: {
+              _: hasError
+                ? "colorBackgroundDestructiveStrong"
+                : "colorBackgroundPrimary",
+              hover: hasError
+                ? "colorBackgroundDestructiveStrong"
+                : "colorBackgroundPrimaryStrong",
+              disabled: {
+                _: "colorBackgroundStrong",
+                hover: "colorBackgroundStrong",
+              },
+            },
+            disabled: {
+              _: "colorBackground",
+              hover: "colorBackground",
+            },
+          }}
+          borderColor={{
+            _: hasError ? "colorBorderError" : "colorBorder",
+            checked: hasError ? "colorBorderError" : "colorBorderPrimary",
+            disabled: "colorBorderWeaker",
+          }}
+          borderRadius="borderRadiusCircle"
+          borderStyle="borderStyleSolid"
+          borderWidth="borderWidth10"
+          disabled={disabled}
+          h="1rem"
+          id={radioID}
+          margin="space0"
+          outlineColor={{ focus: "colorBorderPrimary" }}
+          outlineOffset={{ focus: "borderWidth20" }}
+          outlineStyle={{ focus: "solid" }}
+          outlineWidth={{ focus: "borderWidth10" }}
+          padding="space0"
+          ref={ref}
+          value={value}
+          w="1rem"
+          {...props}
+        >
+          <Box.span
+            alignItems="center"
+            as={RadioIndicator}
+            borderRadius="borderRadiusCircle"
+            display="flex"
+            h="100%"
+            justifyContent="center"
+            w="100%"
+          >
+            <Box.span
+              backgroundColor="colorBackground"
+              borderRadius="borderRadiusCircle"
+              h="0.375rem"
+              w="0.375rem"
+            />
+          </Box.span>
+        </Box.button>
+        <Text.label
+          color={hasError ? "colorTextError" : "colorTextStronger"}
+          fontFamily="fontFamilyModerat"
```

Comment:
I think you don't need to set this since it's part of the baseStyles https://github.com/Localitos/pluto/blob/main/packages/theme/src/styles/base.tsx#L6

Reply:
@blumaa @apvale Let's keep the font family there. Its a safer way to make sure our font family stays Moderat. I.e. in the client app Tailwind has a tendency to over-write our Pluto font styles even though we set it on the base styles. Basically Tailwind can be too heavy handed, so we need to be extra protective of our styles.